com.github.franjegueje.deckico.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.franjegueje.deckico
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: deckico-gui
+finish-args:
+  - --socket=x11
+  - --filesystem=~/.config/autostart
+  - --filesystem=~/.local/share/Steam/steamapps/compatdata
+  - --filesystem=~/.local/share/Steam/steamapps/shadercache
+  - --filesystem=~/.local/share/Steam/userdata:ro
```

Comment:
this will not work for a flatpak installed steam.

Reply:
This flatpak will be used to Steam on local (Steam Deck) not for flatpak installed steam